netbout-client/src/main/java/com/netbout/mock/MkBase.java

```diff
@@ -122,5 +110,26 @@ public Bout randomBout() throws IOException {
         );
         return bout;
     }
+    /**
+     * Create alias.
+     * @param name Name of the alias
+     * @return Alias
+     * @throws IOException If fails
+     */
+    public Alias alias(final String name) throws IOException {
+        final User user = this.user(
+            URN.create(
+                String.format(
+                    "urn:test:%d",
+                    new SecureRandom().nextInt(Integer.MAX_VALUE)
```

Comment:
@pinaf it is my mistake, it should be <code>MkBase.RANDOM</code>, fixed, thank you